README.md

```diff
@@ -40,6 +40,15 @@ Basic package managing utilities are provided in the Conda module:
 
 Parameter `env` is optional and defaults to `ROOTENV`. See below for more info.
 
+## Miniconda Python Version
+There are two versions of Miniconda, one for Python 2, and one for Python 3.
+This controls the initial version of python uses in the setup of Conda itself,
+and thus the default version of python for all dependancies.
+You can change the version by setting the enviroment variable `CONDA_JL_VERSION` to `"3"`, prior to installing the Conda.jl package.
+If you change this after Conda.jl has been added, you will need to rerun `Pkg.build("Conda")`.
```

Comment:
Presumably, yes.
Thinking about this, 
Perhaps I should add more detail on it to the readme, and tell the user to do it.
With warnings.
Since deleting the ROOTENV, will break any packages they are using, that used Conda.jl to install their dependencies.
Which does not sound like something that should happen automatically.